wrapping/src/test_python.py

```diff
@@ -51,11 +53,21 @@
 dipoles = om.Matrix()
 dipoles.load(dipole_file)
 
-sensors = om.Sensors()
+sensors = om.MEGSensors()
 sensors.load(squidsFile)
+sensors.info()
 
-patches = om.Sensors()
+patches = om.EEGSensors()
 patches.load(patches_file)
```

Comment:
since we create new classes here. Would it make sense to be able to pass the filename to the constructor so we would just write:

`patches = om.EEGSensors(patches_file)`

?